examples/getting-started-app/README.md

```diff
@@ -25,61 +25,20 @@ Before you start developing with `cartridge`, you need to install several utilit
 * `gcc` &mdash; `C` compiler (see details [here](https://gcc.gnu.org/))
 * `unzip`
 
-To create your project in a quick and easy manner, install the latest tagged version of `cartridge-cli`
-tool (not `scm-1` version).
-
-To see available versions of cartrdge-cli, say:
-```bash
-you@yourmachine $ tarantoolctl rocks search cartridge-cli
-```
-
-You should see something like that:
-```
-cartridge-cli - Search results for Lua 5.1:
-===========================================
+To create your project in a quick and easy manner, install `cartridge-cli`
```

Comment:
I think we should app repo setup guide with `curl -L https://tarantool.io/release/2/installer.sh | bash`  here for new users just in case.